receiver/windowsperfcountersreceiver/README.md

```diff
@@ -50,63 +61,78 @@ If you would like to scrape some counters at a different frequency than others,
 you can configure multiple `windowsperfcounters` receivers with different
 `collection_interval` values. For example:
 
-```yaml
+```yaml 
 receivers:
   windowsperfcounters/memory:
+    metric_metadata:
+      - metric_name: bytes.committed
+        description: the number of bytes committed to memory
+        unit: By
+        gauge:
+          value_type: int
     collection_interval: 30s
     perfcounters:
       - object: Memory
         counters:
-          - Committed Bytes
+          - name: Committed Bytes
+            metric_name: bytes.committed
 
   windowsperfcounters/processor:
     collection_interval: 1m
+    metric_metadata:
+      - metric_name: processor.time
+        description: active and idle time of the processor
+        unit: "%"
+        gauge:
+          value_type: double
     perfcounters:
       - object: "Processor"
         instances: "*"
         counters:
-          - "% Processor Time"
+          - name: "% Processor Time"
+            metric_name: processor.time
+            attributes:
+              state: active
       - object: "Processor"
         instances: [1, 2]
         counters:
-          - "% Idle Time"
+          - name: "% Idle Time"
+            metric_name: processor.time
+            attributes:
+              state: idle
 
 service:
   pipelines:
     metrics:
       receivers: [windowsperfcounters/memory, windowsperfcounters/processor]
 ```
 
-### Changing metric format
+### Defining metric format
 
-To report metrics in the desired output format, it's recommended you use this
-receiver with the [metrics transform
-processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/metricstransformprocessor).
+To report metrics in the desired output format, build a metric the metric and reference it in the given counter with any applicable attributes.
```

Comment:
```suggestion
To report metrics in the desired output format, define a metric and reference it in the given counter with any applicable attributes.
```

Reply:
The examples are quite thorough, but we should really have a section that describes all the fields in a metric and calls out their default values.

Reply:
I added a table but it's a bit odd considering there is nothing to be defined for gauge but not having an explicit way to define a gauge also seems odd.